src/modules/Search/Search.js

```diff
@@ -166,6 +166,11 @@ export default class Search extends Component {
     /** A search can have its results take up the width of its container. */
     fluid: PropTypes.bool,
 
+    /** A search input can take up the width of its container . */
+    input: PropTypes.shape({
```

Comment:
I've opened https://github.com/Semantic-Org/Semantic-UI-React/pull/1140 to add this missing method.  Sorry about that, didn't realize it was missing!